### DISCUSSION TITLE
Uno for astro

### DISCUSSION THREAD

**Discussion by louiss0** [unocss/unocss#1446] [Ideas]
Can you guys please create an astrojs integration for unocss? [Astrojs ](https://astro.build/) is a web framework for building web applications. It allows you to create plugins for it called [Integrations](https://docs.astro.build/en/guides/integrations-guide/) you use them to extend the framework. Fred schott created a tailwind one that allows you to integrate tailwind into your app by injecting a global style sheet into each page. Can you do the same for windi or unocss

**Comment by zyyv** [comment]
See [unocss/astro](https://github.com/unocss/unocss/blob/main/packages/unocss/src/astro.ts). For more use cases, please check the [example](https://github.com/unocss/unocss/tree/main/examples/astro).
